acmwebsite/templates/index.xhtml

```diff
@@ -9,11 +9,9 @@
   <body py:block="body" py:strip="True">
     <div class="row">
       <div class="col-md-7">
-        <div class="home-banner">
-          <img class="home-banner-img" src="${tg.url('/img/banner.jpg')}" />
-          <small>
-          Image above: ACM Members and other clubs at a Mozilla Campus Clubs training event
-          </small>
+        <div py:if="banner is not None" class="home-banner">
```

Comment:
![2017-12-02-03 42 03](https://user-images.githubusercontent.com/16734772/33514528-df45385c-d712-11e7-85a4-d0de105b45bb.png)

I don't really like the look of this. There is a lot of whitespace to the left of the "Mines ACM Student Chapter" text. @jackrosenthal, what do you think?

If we want the "Mines ACM Student Chapter" text left-aligned when there is no banner image, this `py:if` statement should be on the `div` above this (with class `col-md-7`) so that the "Mines ACM Student Chapter" text is the first `div` inside of this row.

Reply:
Honestly no issues with it ... it is intended that a picture show there, and if there's not, it's nice keeping it consistent with the width of the text (intention is that you would not need to make banners during development).

Maybe toss a bootstrap "jumbotron" in the oppisite column when there is no image? Not very important.

Reply:
How does this look? (I just changed the class of the div on the right from col-md-5 to jumbotron, as @jackrosenthal suggested)

No banner: ![acm-no-banner](https://user-images.githubusercontent.com/22877007/34061962-88a3f110-e1a7-11e7-9e21-c487e4ebd635.png)

With banner: ![acm-with-banner](https://user-images.githubusercontent.com/22877007/34061963-8b069dfe-e1a7-11e7-956b-c0681be8901e.png)

Reply:
@edwargix , I like the look of that overall. I think you need to nest the jumbotron inside of a `col-md-12` so that it doesn't spill over the sides of the page (note that it's wider than the nav bar and the two boxes below).